### PR TITLE
[WNTR - 20] implement db connection

### PR DESCRIPTION
**Pull request by pwdypb**
Not yet finished (not sure if the [WIP] prefix can block the merge or not)

Finished, ready for review.

### CODE REVIEW

**Comment by pwdypb** [comment]
Please note that after this gets merged you'll have to start the app with the dev maven profile in local.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+root = true
```
Comment (sajti12) [inline .editorconfig]: Code conventions could be addressed in a separate issue, not this one.

File: src/main/java/com/artos/mentoring/dao/TravelerDao.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import com.artos.mentoring.model.Traveler;
+import org.springframework.data.repository.CrudRepository;
+
+/**
+ * Created by Benjámin Árva on 2017. 09. 04..
+ */
+public interface TravelerDao extends CrudRepository<Traveler, Long> {
+
+    Traveler findByEmail(String email);
```
Comment (sajti12) [inline src/main/java/com/artos/mentoring/dao/TravelerDao.java:11]: Is email guaranteed to be a unique attribute? I'm not quite sure we should use email as a way to identify users.
If you've just pushed it as a placeholder, then ignore the comment.

File: pom.xml [hunk 3]
```diff
@@ -125,4 +135,39 @@
...
 
+    <profiles>
+        <profile>
+            <id>dev</id>
+            <properties>
+                <spring.profiles.active>dev</spring.profiles.active>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>com.h2database</groupId>
```
Comment (sajti12) [inline pom.xml:146]: What's your take on using this as an embedded db for dev purposes?

File: src/main/java/com/artos/mentoring/dao/TravelerDao.java (hunk 2 above)
Comment (pwdypb) [reply src/main/java/com/artos/mentoring/dao/TravelerDao.java:11]: OK, ignored

Comment (pwdypb) [reply src/main/java/com/artos/mentoring/dao/TravelerDao.java:11]: daaaamn...

Comment (sajti12) [reply src/main/java/com/artos/mentoring/dao/TravelerDao.java:11]: You were quick.

_that's what she said..._

**Review by sajti12 — APPROVED** [review]

**Pull request merged**